src/gen-use-table.py

```diff
@@ -338,9 +341,6 @@ def map_to_use(data):
 		if 0x1CF2 <= U <= 0x1CF3: UIPC = Right
 		if 0x1CF8 <= U <= 0x1CF9: UIPC = Top
 
-		if not (UIPC in [Not_Applicable, Visual_Order_Left] or USE in use_positions):
```

Comment:
Why this change?

Reply:
Oh that was just added, I probably should squash the changes.

Reply:
Yeah, looked at the squashed diff and looks good.